### ISSUE TITLE
v1 vs v2 API

### ISSUE THREAD

**Issue by philfreo** [winfinit/wink-js#8]
This library is for v1 of Wink's API, correct?

Any plans for releasing a v2 version? Is there any reason why it can't be done with v2?

**Comment by winfinit** [comment]
@philfreo that is correct, this library is for v1. I have a request pending for a new keys for v2.

**Comment by philfreo** [comment]
Thanks. Per my PR, the docs are now clarified. Looking forward to seeing a v2 version! Let us know if there are specific things you're looking for help on.
